bigquery/populate_datasets.py

```diff
@@ -1,35 +1,58 @@
-import ponder; ponder.init()
+import ponder
+
+ponder.init()
 from google.cloud import bigquery
 from google.cloud.bigquery import dbapi
 from google.oauth2 import service_account
 import json
 
 db_con = dbapi.Connection(
-            bigquery.Client(
-            credentials=service_account.Credentials.from_service_account_info(
-                    json.loads(open("credential.json").read()),
-                    scopes=["https://www.googleapis.com/auth/bigquery"],
-                )
-            )
+    bigquery.Client(
+        credentials=service_account.Credentials.from_service_account_info(
+            json.loads(open("credential.json").read()),
+            scopes=["https://www.googleapis.com/auth/bigquery"],
         )
+    )
+)
 
-ponder.configure(bigquery_dataset='TEST', default_connection=db_con)
+ponder.configure(bigquery_dataset="TEST", default_connection=db_con)
 
 import modin.pandas as pd
 
-def try_upload_csv(db_con,url,name):
+
+def try_upload_csv(db_con, url, name):
     df = pd.read_csv(url)
     try:
-        df.to_sql(name,db_con,index=False)
+        df.to_sql(name, db_con, index=False)
         print(f"Uploaded dataset to {name}")
     except ValueError as e:
-        print(f"TEST.{name} already exists")
+        print(f"PONDER.{name} already exists")
+
 
 # try_upload_csv(db_con,"https://github.com/ponder-org/ponder-datasets/blob/main/citibike_tutorial.csv?raw=True", "PONDER_CITIBIKE")
-try_upload_csv(db_con,"https://github.com/ponder-org/ponder-datasets/blob/main/books.csv?raw=True", "PONDER_BOOKS")
+try_upload_csv(
```

Comment:
Looks like the linter made some modifications here?

Reply:
These are being made automatically via the IDE reformatting the python code, the content is the same however